### ISSUE TITLE
multi-master installation steps. 

### ISSUE THREAD

**Issue by shivakumarss** [JohnSully/KeyDB#43]
Hello,

We are trying out the evaluation for multi master setup where keydb would be installed in 2 different zones having multi master mode enabled. 

I went through this link provided in the wiki https://github.com/JohnSully/KeyDB/wiki/Multimaster-Support but i was unable to figure out the complete configurations required.

Only parameter which i found is multi-master yes in the configuration, i am not sure how would i configure other master to which the sync would happen.

Could you please help me on this.

Thanks.
Shiva

**Comment by benschermel** [comment]
Please take a look at the documentation site: https://docs.keydb.dev/docs/multi-master where there is an example. If you just have 2 nodes you may want to try out active-rep option.

Cheers,

Ben

**Comment by shivakumarss** [comment]
Thanks a lot Ben, I will use the documentation, some how i didnt find it earlier.

I had started with active-active deployment but my use-case seems to not fit in there, let me tell you what i am trying to solve and what is my expectations. 

- right now i am having a isolated redis setup in each of the zones ( EU, Asia, SG ) .
- Each of them are maintaining some counters which are zone specific.
- Now the requirement is these counters should not be zone specific it should be shared across.
- I am looking for a solution, lets a key "a" can be incr across zones and they in sync.

I have already evaluated active-active where it doesn't seems to solve my problem, here is what i did.

```
servera - define a variable and set it to 1 and incr to 3
serverb - verify its value is 3.
-- disconnect the server by giving the wrong port and restart ----
servera - incr the variable till it reaches 6
serverb - incr the variable till it reaches 6
-- sync -- 
servera - value is 6
serverb - value is 6
```
here my expectation was to have a value of 9 but since the framework uses last write strategy, the variable was synced with value which was written by its latest timestamp.

Is there any way i can change the sync strategy ? 

Here i am assuming multi-master would solve the problem, because as per [documentation](https://docs.keydb.dev/docs/multi-master) it would not drop the database while the sync with master and it would merge read/writes.

Thanks.
Shiva

**Comment by JohnSully** [comment]
> Is there any way i can change the sync strategy ?

Hi @shivakumarss 

I intentionally only started with the most simple synchronization strategy.  CRDTs are a planned feature but its quite complex to implement and will take some time.

**Comment by shivakumarss** [comment]
Hi @JohnSully  Thanks for letting us know. 
I have created another ticket for feature request for CRDT sync strategy https://github.com/JohnSully/KeyDB/issues/47 for us to know once it is done and i am closing this ticket. 

Thanks a lot. 
Shiva
